### ISSUE TITLE
R version 4?

### ISSUE THREAD

**Issue by nbreznau** [Ronggui/RQDA#44]
Just curios if there are plans to make RQDA run on version 4 of R?

**Comment by BroVic** [comment]
Hi @nbreznau, I'm not a maintainer but let me chime in.

**TLDR;**
You can install the current stable RQDA on R 4.0 and it works without problems. See #38.

---

There's a slight misconception some have over RQDA vs R 4.0. The problem we are having with RQDA is not one of backward compatibility but of accessibility. 

RQDA was archived on CRAN due to changes in 2 of its dependencies, _**gWidgets**_ and _**gWidgetsRGtk2**_. Because of this, we can't currently simply install RQDA with the regular call to `install.packages()`.

I am currently using the archived version of RQDA on R-4.0.3, without any problems. The question is how did I install it? Reference to the discussion on issue #38. In light of the problem we discussed at that time, I wrote a script to help with installing it on Windows. See **[this SO question](https://stackoverflow.com/a/65521832/5420406)** where I described it in detail. Also, [Jan Marvin Garbuszus](https://github.com/JanMarvin) wrote developed a branch using the new versions of the problematic RQDA dependencies, and many reported success installing it, but it looks like the repo was either deleted or moved to a different location. I, however, preferred to stick with the CRAN version.

Finally, the maintainer(s) of this project have made new commits that indicate they are working on upgrading the package. I expect it all to be resolved in no time.

**Comment by Lisette218** [comment]
Thanks for all the developer's work to make the package available and running on R 4.0. 

Unfortunately simply installing RQDA currently doesn't seem to work anymore (on Windows). I tried many of the solutions which were mentioned in [#38](https://github.com/Ronggui/RQDA/issues/38), and [this approach](https://github.com/Ronggui/RQDA/issues/38#issuecomment-705305841) seemed to be the most promising, The problem now seems to be related (again) to difficulties with installing the "gWidgetsRGtk2"-package.

I don't know whether more people are experiencing this problem, but I thought it might be worth mentioning.

**Comment by BroVic** [comment]
If it's Windows install [RQDAassist](https://github.com/BroVic/RQDAassist) and run

```
RQDAassist::install()
```

The problem is not with gWidgetsRgtk2 itself, but indirectly RGtk2 and its direct dependency, Gtk+. Gtk+ is not an R package and thus has to be installed independently in the following 2 possible ways:

1. Installed as a standalone software and placed on system PATH, or
2. Installed in the gtk subfolder of the RGtk2 found in the local R library.

For the first instance, you have to download it yourself from the host while in the second, RGtk2 will prompt you to install Gtk+, but only in an interactive R session after a call to  `library(RGtk2)`. Once Gtk+ is present, gWidgetsRgtk2 installs without problems.

**Comment by Lisette218** [comment]
Thanks, works like a charm (after I re-installed R and RStudio)!

**Comment by BroVic** [comment]
Great!

**Comment by MillerGuevara** [comment]
> If it's Windows install [RQDAassist](https://github.com/BroVic/RQDAassist)

Just created this account to say: thank you very much!
I installed RQDA with the code you wrote at https://gist.github.com/BroVic/7771e1e86df35f6410a3f586ea1ef6c6
I'm gonna try this new package tho. Thank you, again.

**Comment by BroVic** [comment]
Glad it worked out for you. 

Yeah I basically used that script to write the main function in that package but then I started adding a few others  to facilitate my own work with RQDA. If you have any suggestions feel free to open an issue over there. Cheers!
